torch/_dynamo/variables/dicts.py

```diff
@@ -611,9 +624,15 @@ def unpack_var_sequence(self, tx):
     def reconstruct(self, codegen: "PyCodegen"):
         # load types.MappingProxyType
         if self.source:
-            unimplemented(
-                "Can't reconstruct an existing mapping variable because"
-                " the connection to the original dict will be lost"
+            msg = (
+                f"MappingProxyVariable {self.source} is not supported because "
+                "can't reconstruct a mapping proxy variable as it will be lost"
```

Comment:
```suggestion
                f"Preexisting MappingProxyVariable (source: {self.source}) cannot be reconstructed "
                "because the connection to the original dict will be lost."
```